### PR TITLE
Moved the apps into Apps/

### PR DESCRIPTION
**Pull request by sjmiles**
also created CommonApp

### CODE REVIEW

**Review by mariakleiner — APPROVED** [review]

File: pkg/Apps/GraphMaker/graph.js [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+        "nodeTypes": [
+          "nodeTypes",
+          "NodeInspectorNode1undefinedinspectorData"
+        ],
+        "categories": [
+          "categories",
+          "NodeInspectorNode1undefinedinspectorData"
+        ],
+        "newNodeInfos": [
+          "NodeInspectorNode1undefinedinspectorData"
```
Comment (mariakleiner) [inline pkg/Apps/GraphMaker/graph.js:29]: `NodeInspectorNode1undefinedinspectorData`
the `undefined` here i think is a bug...

Comment (sjmiles) [reply pkg/Apps/GraphMaker/graph.js:29]: Yes, this is output from Graph!

Comment (mariakleiner) [reply pkg/Apps/GraphMaker/graph.js:29]: unfortunately, i couldn't make GraphMaker work for me, I'm getting a split panel, all empty... we should look at it together (probably in the new year)

Comment (sjmiles) [reply pkg/Apps/GraphMaker/graph.js:29]: GraphMaker (Apps/GraphMaker) is not Graph! (Apps/Graph). GraphMaker is just a façade right now, as we make it possible to describe GraphMaker itself in terms of graphs.

Comment (sjmiles) [reply pkg/Apps/GraphMaker/graph.js:29]: Sorry, yes, GraphMaker is not rendering properly, I probably just broke it, I will check.

Comment (sjmiles) [reply pkg/Apps/GraphMaker/graph.js:29]: I changed all the containers to have 'container' in their slot names, as a simple way to filter the NodeTree. I failed to update `graph.js` for the new names, I believe this is why the GraphMaker looks weird.

**Pull request merged**